2021-09-16 01:21:58,513 INFO    MainThread:15944 [wandb_setup.py:_flush():69] setting env: {}
2021-09-16 01:21:58,514 INFO    MainThread:15944 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-16 01:21:58,514 INFO    MainThread:15944 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-16 01:21:58,514 INFO    MainThread:15944 [wandb_init.py:_log_setup():342] Logging user logs to F:\5th sem project\wandb\run-20210916_012158-2tvcq4w8\logs\debug.log
2021-09-16 01:21:58,514 INFO    MainThread:15944 [wandb_init.py:_log_setup():343] Logging internal logs to F:\5th sem project\wandb\run-20210916_012158-2tvcq4w8\logs\debug-internal.log
2021-09-16 01:21:58,516 INFO    MainThread:15944 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001C48A0C6490>
2021-09-16 01:21:58,539 INFO    MainThread:15944 [wandb_init.py:init():375] calling init triggers
2021-09-16 01:21:58,539 INFO    MainThread:15944 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-16 01:21:58,540 INFO    MainThread:15944 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 2cahq1cy
2021-09-16 01:21:58,540 INFO    MainThread:15944 [wandb_run.py:finish():1271] finishing run khushal/last-weather-recognization/2cahq1cy
2021-09-16 01:21:58,541 INFO    MainThread:15944 [wandb_init.py:init():424] starting backend
2021-09-16 01:21:58,541 INFO    MainThread:15944 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-09-16 01:21:58,548 INFO    MainThread:15944 [backend.py:ensure_launched():135] starting backend process...
2021-09-16 01:21:58,659 INFO    MainThread:15944 [backend.py:ensure_launched():139] started backend process with pid: 220
2021-09-16 01:21:58,660 INFO    MainThread:15944 [wandb_init.py:init():429] backend started and connected
2021-09-16 01:21:58,663 INFO    MainThread:15944 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-09-16 01:21:58,663 INFO    MainThread:15944 [wandb_run.py:_label_probe_notebook():822] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-16 01:21:58,664 INFO    MainThread:15944 [wandb_init.py:init():477] updated telemetry
2021-09-16 01:21:58,665 INFO    MainThread:15944 [wandb_init.py:init():500] communicating current version
2021-09-16 01:22:01,282 INFO    MainThread:15944 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-16 01:22:01,283 INFO    MainThread:15944 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-16 01:22:02,627 INFO    MainThread:15944 [wandb_init.py:init():540] starting run threads in backend
2021-09-16 01:22:07,641 INFO    MainThread:15944 [wandb_run.py:_console_start():1601] atexit reg
2021-09-16 01:22:07,643 INFO    MainThread:15944 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-09-16 01:22:07,643 INFO    MainThread:15944 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-09-16 01:22:07,645 INFO    MainThread:15944 [wandb_run.py:_redirect():1536] Redirects installed.
2021-09-16 01:22:07,645 INFO    MainThread:15944 [wandb_init.py:init():565] run started, returning control to user process
2021-09-16 01:22:07,645 INFO    MainThread:15944 [wandb_config.py:__setitem__():141] config set epochs = 60 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001C48DAECBB0>>
2021-09-16 01:22:07,645 INFO    MainThread:15944 [wandb_run.py:_config_callback():843] config_cb epochs 60 None
2021-09-16 01:22:07,645 INFO    MainThread:15944 [wandb_config.py:__setitem__():141] config set size = 256 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001C48DAECBB0>>
2021-09-16 01:22:07,646 INFO    MainThread:15944 [wandb_run.py:_config_callback():843] config_cb size 256 None
2021-09-16 01:22:07,646 INFO    MainThread:15944 [wandb_config.py:__setitem__():141] config set dropout = 0.0 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001C48DAECBB0>>
2021-09-16 01:22:07,646 INFO    MainThread:15944 [wandb_run.py:_config_callback():843] config_cb dropout 0.0 None
2021-09-16 01:22:07,646 INFO    MainThread:15944 [wandb_config.py:__setitem__():141] config set batch_size = 14 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001C48DAECBB0>>
2021-09-16 01:22:07,646 INFO    MainThread:15944 [wandb_run.py:_config_callback():843] config_cb batch_size 14 None
2021-09-16 01:22:07,647 INFO    MainThread:15944 [wandb_config.py:__setitem__():141] config set pool_size = [7, 7] - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001C48DAECBB0>>
2021-09-16 01:22:07,647 INFO    MainThread:15944 [wandb_run.py:_config_callback():843] config_cb pool_size [7, 7] None
2021-09-16 01:22:07,647 INFO    MainThread:15944 [wandb_config.py:__setitem__():141] config set shear_range = 0.4 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001C48DAECBB0>>
2021-09-16 01:22:07,647 INFO    MainThread:15944 [wandb_run.py:_config_callback():843] config_cb shear_range 0.4 None
2021-09-16 01:22:07,647 INFO    MainThread:15944 [wandb_config.py:__setitem__():141] config set zoom_range = 0.4 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001C48DAECBB0>>
2021-09-16 01:22:07,648 INFO    MainThread:15944 [wandb_run.py:_config_callback():843] config_cb zoom_range 0.4 None
2021-09-16 01:50:26,216 INFO    MainThread:15944 [wandb_run.py:finish():1271] finishing run khushal/last-weather-recognization/2tvcq4w8
